sdk/policyinsights/Microsoft.Azure.Management.PolicyInsights/src/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure policy insights operations.")]
 
 [assembly: AssemblyVersion("3.0.0.0")]
```

Comment:
Also, you really want to change AssemblyVersion - because the types are different, this could result in difficult to detect errors if this assembly were substituted for a previous version of this dll

Reply:
ok, Updated Assembly version as well.